src/main/sql/create.sql

```diff
@@ -1,5 +1,5 @@
-create table "COMPUTERS" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL,"MANUFACTURER_ID" INTEGER NOT NULL);
-create table "COMPANIES" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL);
+create table IF NOT EXISTS "COMPUTERS" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL,"MANUFACTURER_ID" INTEGER NOT NULL);
+create table IF NOT EXISTS "COMPANIES" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL);
```

Comment:
I think so, too. But in my environment when running `;clean; gen-tables`, `java.lang.RuntimeException: org.h2.jdbc.JdbcSQLException: Table "COMPUTERS" already exists; SQL statement:` has occured.
I haven't check what is wrong, just ignore this.